examples/src/main/java/io/dapr/examples/actors/http/README.md

```diff
@@ -108,6 +108,30 @@ cd to [repo-root]
 dapr run --app-id demoactorservice --app-port 3000 --port 3005 -- mvn exec:java -pl=examples -D exec.mainClass=io.dapr.examples.actors.http.DemoActorService -D exec.args="-p 3000"
 ```
 
+### Debugging the Demo actor service
+
+If you want to debug the `Subscriber`, you have to make sure to provide the port as an argument.
+
+For VSCode you can find a sample launch.json which includes:
+```json
+...
+{
+    "type": "java",
+    "name": "Debug (Launch)-DemoActorService<dapr-sdk-examples>",
+    "request": "launch",
+    "mainClass": "io.dapr.examples.actors.http.DemoActorService",
+    "projectName": "dapr-sdk-examples",
+    "args": "-port=3000"
+},
+...
+```
+
+Use the following command to run the Dapr sidecar:
+
+```sh
+dapr run --app-id demoactorservice --app-port 3000 --port 3005 --grpc-port 5001 -- waitfor FOREVER
```

Comment:
waitfor is not a builtin command in shell. users needs to install it separately.

Reply:
I just realized after the PR that at each debugging doc we need to separate between Unix and Windows if we go with ```dapr``` and dummy commands.
Thus my suggestion is to change it to ```daprd``` - see #212

Reply:
@RicardoNiepel for now, let's use `cat` for linux and mac. please add additional command in this doc. if we use daprd, then we may need to update the existing docs. Rather than this, I make dapr cli to run alone without app. 

Also Are you supposed to use vscode even for java?

Reply:
We're using `waitfor` on Windows and `cat` on MacOS/Linux just to have a dummy command so the user can start their app separately through an IDE/debugger.  These two options are listed in the README.md in the root of the repo ie

```
 For Linux and MacOS:

dapr run --app-id testapp --app-port 3000 --port 3500 --grpc-port 5001 -- cat
For Windows:

dapr run --app-id testapp --app-port 3000 --port 3500 --grpc-port 5001 -- waitfor FOREVER
```


We used `cat` on MacOS/Linux but unfortunately it's not built-in on Windows, so we used `waitfor`, which is built into Win7 and later.

In places where the command differs between OSes, we've tried to list the command to run per OS.  It would be good to list the commands per OS like the text I pasted above.

Reply:
There is significant logic in Dapr cli before it can invoke daprd with the right arguments.

Reply:
I will do this